--- tclvfs/doc/vfslib.n.org	2023-12-09 20:52:35.019316238 +0800
+++ tclvfs/doc/vfslib.n	2023-12-09 20:52:43.351327702 +0800
@@ -2,7 +2,6 @@
 '\" Copyright (c) 2001, Vince Darley
 '\"
 '\"
-.so man.macros
 .TH vfslib n 1.0 Vfslib "Tcl-only Virtual File Systems"
 .BS
 '\" Note:  do not modify the .sh NAME line immediately below!
--- tclvfs/doc/vfs.n.org	2023-12-09 20:53:00.971351933 +0800
+++ tclvfs/doc/vfs.n	2023-12-09 20:53:08.183361859 +0800
@@ -2,7 +2,6 @@
 '\" Copyright (c) 2001-2003, Vince Darley
 '\"
 '\"
-.so man.macros
 .TH vfs n 1.4 Vfs "Tcl-only Virtual File Systems"
 .BS
 '\" Note:  do not modify the .sh NAME line immediately below!
